assignments/week_5/week_5_exercises/exercise_5_1/exercise_5_1.md

```diff
@@ -0,0 +1,40 @@
+# Exercise 5.1
+This exercise covers how to create a launch file for node.
+
+# Exercise Objective
+Create a `.launch` file which starts the `printing_node`, which is already created for you.
+
+A launch file structured with XML (like HTML): it is composed of tags, which have attributes and 
+can contain variables and other tags.
+
+# Structure of a launch file
+Every launch file needs `<launch>` tags surrounding all it's contents:
+
+```xml
+<launch>
+
+</launch>
+```
+
+To make a launch file start node, you need a `<node>` tag inside the `<launch>` tags.
+The node tag requires three attributes:
+* `pkg`: which package the node can be found in
+* `type`: the type of node to launch, this is the string value put in `ros::init`
+* `name`: the name of this individual node. There can be multiple nodes with the same type,
+but no two nodes can have the same name.
+
+There is also an optional attribute `output`, which determines where the node should
+print messages. For this exercise, you want `output="screen"`
```

Comment:
I would have them run the node without output="screen" and with to get the idea of what is printed with each and what is not.